config/application.yml.default

```diff
@@ -43,6 +43,8 @@ asset_host:
 async_wait_timeout_seconds: '60'
 available_locales: en es fr
 aws_http_timeout: '5'
```

Comment:
This is the most difficult number to set, but also the biggest lever in avoiding the Rack timeout. A couple services hit the current maximum at least once a day from what I'm seeing in NewRelic, so I'm wary of reducing it across the board.